### ISSUE TITLE
How to get the Last layer (ctc_greedy_decoder)?

### ISSUE THREAD

**Issue by danishansari** [FLming/CRNN.tf2#35]
Hi, nice work!!
The model provided here for demo (SavedModel) has the last layer **ctc_greedy_decoder**, while the model in train.py this layer is not present. How do we get it? is this layer added after training if so can you please tell, how to get it done?

> logits (Dense)               (None, None, 38)          19494     
> _________________________________________________________________
> ctc_greedy_decoder (CTCGreed ((None,), (None,))        0

**Comment by danishansari** [comment]
Okay, got it. Its done using export.py. Which is present in **crnn** directory and not in tools, which is mentioned in README.
